src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_TupleCreationExpression.cs

```diff
@@ -59,7 +56,18 @@ private BoundNode RewriteTupleCreationExpression(BoundTupleCreationExpression no
                                                                                               TupleTypeSymbol.RestPosition - 1)
                                                                                       .Add(currentCreation);
 
-                        MethodSymbol constructor = tuple8Ctor.AsMember(underlyingTupleTypeChain.Pop());
+                        NamedTypeSymbol currentType = underlyingTupleTypeChain.Pop();
+                        var tuple8Ctor = (MethodSymbol)TupleTypeSymbol.GetWellKnownMemberInType(currentType.OriginalDefinition,
+                                                                                                TupleTypeSymbol.GetTupleCtor(TupleTypeSymbol.RestPosition),
+                                                                                                _compilation.Assembly,
+                                                                                                _diagnostics,
+                                                                                                node.Syntax);
+                        if ((object)tuple8Ctor == null)
+                        {
+                            return node;
+                        }
```

Comment:
The reason I pulled it into the loop is I worried about a mixed chain of two kinds of ValueTuples, where the different elements popped have the same arity but are not the same type. I don't know if we could ever hit that though.
Since the change is to do all ValueTuple member lookups relative to the underlying type, I didn't want to take a chance.